rnd/autogpt_server/migrations/20240902223334_add_stats_column/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AlterTable
```

Comment:
not one for your PR but we should remove these and move the postgres ones up

Reply:
So the default migration is postgres and sqlite migration is removed ?

Reply:
@aarushik93 let me know if you should rebase this or I should do it.